### PR TITLE
chore: adding AWS link and Add GCP to the Kafka overview page

### PR DESCRIPTION
**Pull request by Kells512**
**What this PR does / why we need it**:

>Making updated to the Kafka Overview page, and making version 3. 
>Changes found in document: https://docs.google.com/document/d/1zhkNbi5pI2_tc2USqbkcm0MFnuOKg97TvtEWwiEUErk/edit

**Verification steps** 

> APPSERVICES > <Kafka Page V 3> 

> Updating Purchase now and Contact sales cards
>**MGDSTRM-9141: add link to AWS Marketplace to Kafka Overview page:** https://issues.redhat.com/browse/MGDSTRM-9141

Screenshot from 2022-07-14 16-44-45

>Adding GCP to Cloud Providers and make layout more responsive at smaller viewpoints
>**MGDSTRM-9119: Add GCP to the Kafka overview page:** https://issues.redhat.com/browse/MGDSTRM-9119

> **With Vertical dividers and rows**
Screenshot from 2022-07-15 09-14-02

> **With Horizontal dividers and columns**
Screenshot from 2022-07-18 09-19-04

### CODE REVIEW

File: src/AppServices/KafkaPageV3.tsx [hunk 1]
```diff
@@ -0,0 +1,327 @@
...
+                    </Trans>
+                  </Title>
+                </CardTitle>
+              </CardHeaderMain>
+            </CardHeader>
+            <CardBody>{t("purchaseCardMainText")}</CardBody>
+            <CardFooter>
+              <Flex>
+                <FlexItem>
+                  <Button
```
Comment (riccardo-forina) [inline src/AppServices/KafkaPageV3.tsx:74]: I think all these buttons could be converted to use the `ExternalLink` component.

Comment (Kells512) [reply src/AppServices/KafkaPageV3.tsx:74]: Yeah, I'll do that

File: src/AppServices/KafkaPageV3.tsx [hunk 2]
```diff
@@ -0,0 +1,335 @@
...
+                      components={{
+                        bold: <strong />,
+                        size: <span className="pf-u-font-size-lg" />,
+                      }}
+                    />
+                  </ExternalLink>
+                </FlexItem>
+
+                <FlexItem>
+                  <ExternalLink
```
Comment (jgiardino) [inline src/AppServices/KafkaPageV3.tsx:90]: With the current spacing between these buttons, the icon is equally spaced between the text. Ideally, it should be closer to the text label of the link it belongs to, and further from the text in the other link.
image

There's an example of how to modify the space between flex item here: https://www.patternfly.org/v4/layouts/flex#spacing-xl

I suggest using 2xl for this one. Here's an example using 2xl spacing:
image

File: locales/en/kafkaoverview-v3.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  "kafkaInstanceCapacityDescription": "Streams for Apache $t(common:kafka) offers instances in the following sizes with the following capacity.",
+  "kafkaInstanceCapacityfooter": "*Storage can be extended. If you wish to increase the storage limit for a $t(common:kafka) instance, contact <0>Red Hat support</0> to discuss your options. You can check your current storage usage by <1>monitoring disk space metrics</1>.",
+  "kafkaInstanceCapacityTitle": "$t(common:kafka) instance capacity",
+  "microsoftAzureDescription": "Coming soon",
+  "microsoftAzureTitle": "Microsoft Azure",
+  "pricingModalTitle": "Pricing model",
+  "purchaseCardCallToActionButtonAWS": "<size><bold>AWS Marketplace</bold><size>",
+  "purchaseCardCallToActionButtonRedHat": "<size><bold>Red Hat Marketplace</bold><size>",
+  "purchaseCardMainText": "Purchase a pay-as-you-go subscription for up to 25 streaming units using one of our Marketplace options below.",
+  "purchaseCardTitle": "Purchase now <0>(US and Canda Only)</0>",
```
Comment (jgiardino) [inline locales/en/kafkaoverview-v3.json:24]: Instead of this:
> (US and Canda Only)

It should be this:
> (US and Can**a**da **o**nly)

**Review by jgiardino — CHANGES_REQUESTED** [review]
Your updates look great! I noted just a couple of minor updates that I commented on inline.

**Pull request closed without merging**